vrecord

```diff
@@ -254,6 +254,7 @@ _set_lang_variables(){
     elif [[ "${system_language}" == 'zh' ]] ; then
         card_var='card'
     fi
+    _include_var card_var
```

Comment:
I guess this should read:

```bash
    _include_var "${card_var}"
```

Reply:
I didn't write that [particular function](https://github.com/amiaopensource/vrecord/blob/main/vrecord#L124) but after testing it out, found that indeed using `var` instead of `${var}` is how it was designed:
`var='test' ; _include_var var` will pass `var="test"`to the config file, while 

`var='test' ; _include_var "$var"` will pass `test=""`

Reply:
Interesting indeed, thank you!